src/ucs/sys/event_set.h

```diff
@@ -20,7 +20,8 @@ typedef struct ucs_sys_event_set ucs_sys_event_set_t;
  * ucs_event_set_handler call this handler for notifying event
  *
  */
-typedef void (*ucs_event_set_handler_t)(int fd, int event, void *arg);
+typedef void (*ucs_event_set_handler_t)(void *callback_data, int event,
```

Comment:
pls add documentation of callback arguments

Reply:
Added doxygen comments.

```C
/**
 * ucs_event_set_handler call this handler for notifying event
 *
 * @param [in] callback_data  User data which set in ucs_event_set_add().
 * @param [in] event          Detection event. Sets of ucs_event_set_type_t.
 * @param [in] arg            User data which set in ucs_event_set_wait().
 *
 */
```